source/hardware/controllers/korg_kaoss_dj.rst

```diff
@@ -42,15 +42,15 @@ Mapping
-| 8                                              | Tap Button                                     | Sets the tempo                                 | Key setting mode                               |
-|                                                |                                                | Long-press this button to access the auto BPM  |                                                |
-|                                                |                                                | function                                       |                                                |
+| 8                                              | Tap Button                                     | tap to open a folder in the file-browser       | use left / right shift to set the tempo        |
+|                                                |                                                | (double-tap to close an open folder)           | of left / right track                          |
+|                                                |                                                |                                                |                                                |
 +------------------------------------------------+------------------------------------------------+------------------------------------------------+------------------------------------------------+
 | 9                                              | Hold Button                                    | Enables/disables the touchpad’s hold function  | Scale setting mode                             |
 +------------------------------------------------+------------------------------------------------+------------------------------------------------+------------------------------------------------+
-| 10                                             | Touchpad (controller mode)                     | Controls the effects of the DJ software. The   | Adjusts the Beats Multiplier (TODO)            |
-|                                                |                                                | vertical axis controls the mix (dry/wet) knob  |                                                |
-|                                                |                                                | and the horizontal axis controls the super     |                                                |
+| 10                                             | Touchpad (controller mode)                     | Controls the effects of the DJ software. The   | Controls Quick-Effect super knobs              |
+|                                                |                                                | vertical axis controls the mix (dry/wet) knob  | the horizontal-axis controls deck 1            |
+|                                                |                                                | and the horizontal axis controls the super     | the vertical-axis controls deck 2              |
```

Comment:
If you're touching this unrelated lines, please try to boil down the text
`Controls the effects of the DJ software. The vertical axis controls the mix (dry/wet) knob and the horizontal axis controls the super knob.`
to something like
```
Controls effect unit 1.
Y axis: Mix knob (dry/wet)
X axis: Super knob.
```